diff a/drivers/net/wireless/mediatek/mt7601u/phy.c b/drivers/net/wireless/mediatek/mt7601u/phy.c
--- a/drivers/net/wireless/mediatek/mt7601u/phy.c
+++ b/drivers/net/wireless/mediatek/mt7601u/phy.c
@@ -594,7 +594,7 @@ static void mt7601u_rxdc_cal(struct mt7601u_dev *dev)
 
 void mt7601u_phy_recalibrate_after_assoc(struct mt7601u_dev *dev)
 {
-	mt7601u_mcu_calibrate(dev, MCU_CAL_DPD, dev->curr_temp);
+	// mt7601u_mcu_calibrate(dev, MCU_CAL_DPD, dev->curr_temp);
 
 	mt7601u_rxdc_cal(dev);
 }
@@ -1161,12 +1161,12 @@ static int mt7601u_init_cal(struct mt7601u_dev *dev)
 	ret = mt7601u_mcu_calibrate(dev, MCU_CAL_TXIQ, 0);
 	if (ret)
 		return ret;
-	ret = mt7601u_mcu_calibrate(dev, MCU_CAL_RXIQ, 0);
-	if (ret)
-		return ret;
-	ret = mt7601u_mcu_calibrate(dev, MCU_CAL_DPD, dev->dpd_temp);
-	if (ret)
-		return ret;
+	//ret = mt7601u_mcu_calibrate(dev, MCU_CAL_RXIQ, 0);
+	//if (ret)
+	//	return ret;
+	//ret = mt7601u_mcu_calibrate(dev, MCU_CAL_DPD, dev->dpd_temp);
+	//if (ret)
+	//	return ret;
 
 	mt7601u_rxdc_cal(dev);


diff a/drivers/net/wireless/mediatek/mt7601u/usb.c b/drivers/net/wireless/mediatek/mt7601u/usb.c
--- a/drivers/net/wireless/mediatek/mt7601u/usb.c
+++ b/drivers/net/wireless/mediatek/mt7601u/usb.c
@@ -125,8 +125,8 @@
 
 void mt7601u_vendor_reset(struct mt7601u_dev *dev)
 {
-	mt7601u_vendor_request(dev, MT_VEND_DEV_MODE, USB_DIR_OUT,
-			       MT_VEND_DEV_MODE_RESET, 0, NULL, 0);
+	//mt7601u_vendor_request(dev, MT_VEND_DEV_MODE, USB_DIR_OUT,
+	//		       MT_VEND_DEV_MODE_RESET, 0, NULL, 0);
 }
 
 u32 mt7601u_rr(struct mt7601u_dev *dev, u32 offset)


